lib/media/track/trackpublication.js

```diff
@@ -61,6 +61,21 @@ class TrackPublication extends EventEmitter {
  * @event TrackPublication#trackEnabled
  */
 
+/**
+ * A {@link LocalAudioTrackPublication} or a {@link RemoteAudioTrackPublication}.
+ * @typedef {LocalAudioTrackPublication|RemoteAudioTrackPublication} AudioTrackPublication
+ */
+
+/**
+ * A {@link LocalDataTrackPublication} or a {@link RemoteDataTrackPublication}.
+ * @typedef {LocalDataTrackPublication|RemoteDataTrackPublication} DataTrackPublication
+ */
+
+/**
+ * A {@link LocalVideoTrackPublication} or a {@link RemoteVideoTrackPublication}.
+ * @typedef {LocalVideoTrackPublication|RemoteVideoTrackPublication} VideoTrackPublication
+ */
```

Comment:
Cool 👍 so these will be used by the Participant interface?

Reply:
@markandrus Yes.